crates/uv-workspace/src/pyproject.rs

```diff
@@ -432,6 +432,37 @@ pub struct ToolUv {
     )]
     pub override_dependencies: Option<Vec<uv_pep508::Requirement<VerbatimParsedUrl>>>,
 
+    /// Excludes to apply when resolving the project's dependencies.
+    ///
+    /// Excludes are used to prevent installation of a specific package, even if other dependencies
+    /// require it.
+    ///
+    /// Including a package as an exclude will _not_ trigger installation of the package on its
+    /// own; instead, the package must be requested elsewhere in the project's first-party or
+    ///  transitive dependencies.
```

Comment:
This seems confusing? Is this text copied from the constraints documentation? I don't think we need to explain that excluded dependencies are not installed.